src/Tizen.Multimedia.Vision/MediaVision/InferenceModelConfiguration.cs

```diff
@@ -318,6 +323,7 @@ public InferenceBackendType Backend
         /// </remarks>
         /// <exception cref="ArgumentException"><paramref name="value"/> is not valid.</exception>
         /// <since_tizen> 6 </since_tizen>
+        [Obsolete("Please do not use! This will be deprecated in level 8.")]
```

Comment:
Please change all Obsolete attributes to `[Obsolete("Deprecated since API8; Will be removed in API10.")]` or if it is possible please add the sentence about the alternative suggested method or property `[Obsolete("Deprecated since API8; Will be removed in API10. Please use XXX instead.")]`

Reply:
I updated the description.